pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -468,22 +466,22 @@ func (r *ReferenceAdapter) createCredentials() (gcputil.OperationResult, error)
 		return gcputil.ContinueProcessing()
 	}
 
+	r.logger.Info("Creating credentials")
 	serviceAccount, err := r.gcpClient.GetServiceAccount(osdServiceAccountName)
 	if err != nil {
 		if matchesNotFoundError(err) {
-			r.logger.V(int(logtypes.ProjectReference)).Info("Service Account not yet fully initialized. Retrying in 30 seconds.")
+			r.logger.V(2).Info("Service Account not yet fully initialized. Retrying in 30 seconds.")
```

Comment:
```suggestion
			r.logger.V(1).Info("Service Account not yet fully initialized. Retrying in 30 seconds.")
```

Reply:
without V() is V(0) which means display in prod